src/aliases.jl

```diff
@@ -58,6 +59,7 @@ function number_of_patches end
 @alias number_of_vertices n_vertices
 
 @alias n_partitions number_of_partitions
+@alias n_weak_compositions number_of_weak_compositions
 @alias n_patches number_of_patches
```

Comment:
```suggestion
@alias n_patches number_of_patches
@alias n_weak_compositions number_of_weak_compositions
```
keep the list sorted